src/oas3/transformers/callbacks.ts

```diff
@@ -0,0 +1,28 @@
+import { IHttpCallbackOperation } from '@stoplight/types';
+import { CallbacksObject } from 'openapi3-ts';
+import { transformOas3Operation } from '../operation';
+
+export function translateToCallbacks(callbacks?: CallbacksObject): IHttpCallbackOperation[] | undefined {
```

Comment:
Isn't a callback just an HTTP operation basically? Why create a new type?

Just a question, I know this is a draft, but glad we are thinking about supporting it.

Reply:
Yeah, callback is an operation that is wrapped in an object with callback names as keys (see `myWebhook` here https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#callback-object-example).

My idea is to extend HttpOperation with callback name to achieve two goals:
- keep callbacks in array
- keep callback names for further use

Type is defined here: https://github.com/stoplightio/types/pull/54/files#diff-6b2dc2c69dd576eb785765cc8d3bb904R44

Btw. I'm working on webhooks https://github.com/stoplightio/prism/issues/331